app/src/main/java/com/google/samples/apps/sunflower/viewmodels/PlantListViewModel.kt

```diff
@@ -28,27 +28,19 @@ import com.google.samples.apps.sunflower.data.PlantRepository
  * The ViewModel for [PlantListFragment].
  */
 class PlantListViewModel internal constructor(
-    private val plantRepository: PlantRepository
+    plantRepository: PlantRepository
```

Comment:
super nit: Without the "private val", the class header fits within 100 characters:

`class PlantListViewModel internal constructor(plantRepository: PlantRepository) : ViewModel() {`

Maybe update this so it is just on a single line?